src/css/common.css

```diff
@@ -114,14 +115,16 @@ body {
   height: 15%;
 }
 
-.shoko-scrollbar {
-  scrollbar-color: theme('colors.slider-color') theme('colors.slider-background');
-  scrollbar-width: thin;
-}
+@-moz-document url-prefix() {
```

Comment:
possibly needs explanation on what `@-moz-document url-prefix()` actually does as it is not obvious, i.e. makes these styles only apply in mozilla.